qupath-core-processing/src/main/java/qupath/imagej/detect/cells/ObjectMeasurements.java

```diff
@@ -39,7 +39,7 @@
  * @author Pete Bankhead
  *
  */
-class ObjectMeasurements {
+public class ObjectMeasurements {
```

Comment:
This is so that we can create cell measurements from places other than the WatershedCellDetection Class